lib/time-entries.js

```diff
@@ -35,38 +37,94 @@ class TimeEntries {
    * @memberof TimeEntries
    */
   async create(time_entry) {
+    if (!time_entry || !Object.prototype.hasOwnProperty.call(time_entry, 'workspace_id')) {
+      throw new Error('The parameters must include workspace_id');
+    }
+    if (!time_entry || !Object.prototype.hasOwnProperty.call(time_entry, 'start')) {
+      throw new Error('The parameters must include start');
+    }
     return await this.client.post(`workspaces/${time_entry.workspace_id}/time_entries`, {
-      created_with: 'node-toggle-client',
+      created_with: 'saintedlama/toggl-client',
```

Comment:
@saintedlama Does this change look okay to you as an update for the `created_with` parameter?